sky/services/editing/editing.mojom

```diff
@@ -48,3 +48,13 @@ interface Keyboard {
   Show();
   Hide();
 };
+
+struct ClipData {
+  string text;
+};
+
+[ServiceName="editing::Clipboard"]
+interface Clipboard {
+  SetClipData(ClipData clip);
+  GetClipData() => (ClipData? clip);
```

Comment:
You probably want to add a `string format` argument to this function that lets the client say what sort of data it wants back from the clipboard (e.g., text, images, etc).  See https://developer.mozilla.org/en-US/docs/Web/API/DataTransfer/getData